app/ui/src/app/import-export/import/import.reducer.ts

```diff
@@ -0,0 +1,142 @@
+import { ActionReducerMap, createFeatureSelector } from '@ngrx/store';
+
+import { BaseReducerModel, PlatformStore } from '@syndesis/ui/platform';
+import { IntegrationImportState, IntegrationImportRequest, IntegrationUploadRequest } from './import.models';
+import {
+  IntegrationImportActions,
+  IntegrationFetchComplete,
+  IntegrationFetchFail,
+  IntegrationUpload,
+  IntegrationUploadComplete,
+  IntegrationUploadCancel,
+  IntegrationImport,
+  IntegrationImportComplete,
+  IntegrationImportFail,
+} from './import.actions';
+
```

Comment:
`import * as ImportActions from './import.actions';`is simpler and more scalable, so you won't have to import each new action yo create in the future.